plugins/modules/cluster_info.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Ansible Cloud Team (@ansible-collections)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: cluster_info
+short_description: Gathers information about one or more clusters
+description:
+    - >-
+      Gathers information about one or more clusters.
+      You can search for clusters based on the cluster name, datacenter name, or a combination of the two.
+author:
+    - Ansible Cloud Team (@ansible-collections)
+
+options:
+    cluster:
+        description:
+            - The name of the cluster on which to gather info.
+            - At least one of O(datacenter) or O(cluster) is required.
+        type: str
+        required: false
+        aliases: [cluster_name, name]
+    datacenter:
+        description:
+            - The name of the datacenter.
+            - At least one of O(datacenter) or O(cluster) is required.
+        type: str
+        required: false
+        aliases: [datacenter_name]
+    gather_tags:
```

Comment:
Community module has `show_tag` with a little bit of another meaning. Did you replace it with `gather_tags` on purpose?

Reply:
yes, we used `gather_tags` in the `guest_info` module so I used that same attribute here. The functionality is the same